### DISCUSSION TITLE
How can Whisper capture fast speech correctly?

### DISCUSSION THREAD

**Discussion by toanhuynhnguyen** [openai/whisper#2386] [Q&A]
I have an audio file, and a speaker speaks too fast. And Whisper captures sentences incorrectly.
How can Whisper capture fast speech correctly?
Thank you.

**Comment by misutoneko** [comment]
You can try slowing it down with ffmpeg:
`ffmpeg -hide_banner -i myfastfile.wav -filter:a "atempo=0.75" -vn slow.wav`

Timestamps will need to be adjusted accordingly, but at least you can see if the problem is caused by the speed, or if it's something else.
